client/public/index.html

```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    <link rel="shortcut icon" href="./favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <link rel="stylesheet" href="index.css" type="text/css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Assistant|Kaushan+Script"
+      rel="stylesheet"
     />
-    <link rel="stylesheet" href="index.css" type="text/css">
-    <link href="https://fonts.googleapis.com/css?family=Assistant|Kaushan+Script" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet" />
```

Comment:
Not related to this PR, I've just noticed there's two links to Kaushan script. I don't think the second one's needed.

Reply:
Yup, removed the second one